coral/src/app/features/topics/BrowseTopics.tsx

```diff
@@ -8,11 +8,17 @@ import { useGetTeams } from "src/app/features/topics/hooks/teams/useGetTeams";
 import SelectEnv from "src/app/features/topics/components/select-env/SelectEnv";
 import { useState } from "react";
 import { Flexbox, FlexboxItem } from "@aivenio/design-system";
+import { useSearchParams } from "react-router-dom";
+
+// Use a UUID value to represent empty option value.
+const ALL_TEAMS_VALUE = "f5ed03b4-c0da-4b18-a534-c7e9a13d1342";
+const ALL_ENVIRONMENTS_VALUE = TopicEnv.ALL;
```

Comment:
Should we also use a UUID for the All env option? In my next PR the `TopicEnv` enum will be removed, because it's was not a good idea to add the typing 😬  and if we use a UUID here too, it's more consistent.

Reply:
Lets do that as a trivial followup after: https://github.com/aiven/klaw/pull/225. Also, the API expects `ALL` to be sent in the API request whereas the team query parameter should be omitted when fetching topics for all teams.